### PR TITLE
feat: Add theme inputs

### PR DESCRIPTION
**Pull request by MoizMasud**
Closes https://github.com/carbon-design-system/carbon-components-angular/issues/1999

### CODE REVIEW

File: src/tiles/clickable-tile.component.ts [hunk 1]
```diff
@@ -33,6 +35,11 @@ import { Router } from "@angular/router";
 	</a>`
 })
 export class ClickableTile {
+	@HostBinding("class.bx--tile--light") get lightThemeEnabled() {
+		return this.theme === "light";
+	}
```
Comment (Akshat55) [inline src/tiles/clickable-tile.component.ts:40]: Do we need to assign the class to the host or the anchor link? Class is being assigned to the host & anchor link (see line 28).

Pretty sure you do not need to assign to the host.

Comment (Akshat55) [reply src/tiles/clickable-tile.component.ts:40]: Also if you don't need hostbinding after, please remove host import in line 7

File: src/tiles/expandable-tile.component.ts [hunk 2]
```diff
@@ -39,6 +43,12 @@ export interface ExpandableTileTranslations {
 	`
 })
 export class ExpandableTile implements AfterContentInit {
+	@HostBinding("class.bx--tile--light") get lightThemeEnabled() {
+		return this.theme === "light";
+	}
```
Comment (Akshat55) [inline src/tiles/expandable-tile.component.ts:48]: Same thing as the clickable-tile here. Why do we need to assign the `bx--tile--light` class to the host & the button?

File: src/tiles/selection-tile.component.ts [hunk 3]
```diff
@@ -45,6 +49,13 @@ export class SelectionTile implements AfterViewInit {
 	/**
 	 * The unique id for the input.
 	 */
+
+	@HostBinding("class.bx--tile--light") get lightThemeEnabled() {
+		return this.theme === "light";
+	}
```
Comment (Akshat55) [inline src/tiles/selection-tile.component.ts:55]: Same as previous comments.

**Review by Akshat55 — CHANGES_REQUESTED** [review]

**Review by Akshat55 — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**